integration-test/build.gradle

```diff
@@ -9,6 +9,7 @@ dependencies {
     implementation project(':core')
     implementation project(':schema-loader')
     implementation "com.google.guava:guava:${guavaVersion}"
+    implementation "org.slf4j:slf4j-api:${slf4jVersion}"
```

Comment:
Maybe we also need to add some `slf4j` logger implementation like `org.slf4j:slf4j-simple`?

Reply:
Good catch! I think so. Actually I saw some log messages, but I guess `org.slf4j:slf4j-simple` came from `schema-loader`... I'll explicitly add the dependency here.